WORKING_GROUPS.md

```diff
@@ -81,17 +81,22 @@ Its responsibilities are:
 * Creates and manages build-containers.
 
 
-### [Tracing](https://github.com/nodejs/tracing-wg)
+### [Diagnostics](https://github.com/nodejs/diagnostics)
 
-The tracing working group's purpose is to increase the
-transparency of software written in Node.js.
+The diagnostics working group's purpose is to surface a set of comprehensive,
+documented, and extensible diagnostic interfaces for use by
+Node.js tools and JavaScript VMs.
 
 Its responsibilities are:
-* Collaboration with V8 to integrate with `trace_event`.
-* Maintenance and iteration on AsyncWrap.
-* Maintenance and improvements to system tracing support (DTrace, LTTng, etc.)
-* Documentation of tracing and debugging techniques.
-* Fostering a tracing and debugging ecosystem.
+
+* Collaborate with V8 to integrate `v8_inspector`.
+* Collaborate with V8 to integrate `trace_event`.
+* Maintain and improve `async_wrap` and `async_hooks`.
```

Comment:
So far implementing `async_wrap` has been monitored by the Diagnostic WG. We have maintained the unofficial documentation, issue lists and I/Trevor provides monthly status updates. We also have quite a few APM members, who provides us with usecase information though the WG meetings and issues.

Once `async_hooks` lands we may want to revisit that.